web/themes/custom/bglobal/bglobal.info.yml

```diff
@@ -3,7 +3,7 @@ type: theme
 description: 'Berkeley Global D9 theme.'
 package: Core
 core_version_requirement: ^8 || ^9
-base theme: stark
+base theme: stable
```

Comment:
Make sure you check `core.extension.yml` and make sure stark is disabled and stable is enabled, then.

Reply:
Actually nothing is enabled right now:
```
theme:
  bartik: 0
  seven: 0
  bglobal: 0
  stark: 0
```
The base theme change should not be a breaking change.